circle.yml

```diff
@@ -1,7 +1,12 @@
 machine:
   ruby:
     version: 2.3.3
+database:
+  override:
+    - ls &>/dev/null
```

Comment:
Not sure about this line. Because of engine_cart, you have to bypass circle's default database creation routines. There isn't much out there that I could find on doing that except inserting an overwrite command. This line just does something dumb to fill space because leaving it blank doesn't work.